index.bs

```diff
@@ -474,8 +474,22 @@ An implementation may define <dfn export>extension modules</dfn>. These must hav
 [=module name=] that contains a single colon "<code>:</code>" character. The
 part before the colon is the prefix; this is typically the same for all
 [=extension modules=] specific to a given implementation and should be unique for a
-given implementation. Such modules extend the [=local end definition=] and [=remote end definition=]
-providing additional groups as choices for the defined [=commands=] and [=events=].
+given implementation.
+Such modules extend the [=local end definition=] and [=remote end definition=]
+providing additional groups as choices for the defined [=commands=], [=events=] and types.
+
+<dfn export>External specifications</dfn> may define their own modules that
+extend the protocol.
+The recommendation is that such modules should have a [=module name=] that does
+not contain a colon "<code>:</code>" character, and that it is preferred
+to add new modules or commands rather than extending existing ones.
+Such modules may extend the [=local end definition=] and [=remote end definition=] providing
+additional groups as choices for the defined [=commands=], [=events=] and types.
+
+When defining [=extension modules=], implementers should consider the
```

Comment:
I think this paragraph should be removed because it is in conflict with the naming constraint of the implementation extension modules.

Reply:
I agree, this was an oversight, thought I had removed it. Done.